cmd/crank/beta/validate/cache.go

```diff
@@ -41,6 +42,7 @@ type Cache interface {
 type LocalCache struct {
 	fs       afero.Fs
 	cacheDir string
+	w        io.Writer
```

Comment:
the writer here sounds odd, it's only used as a logger and only in the Flush, what if we moved the print to the caller and avoided passing it in altogether?